lib/cocoapods/installer.rb

```diff
@@ -33,6 +33,7 @@ class Installer
     autoload :FileReferencesInstaller,  'cocoapods/installer/file_references_installer'
     autoload :PostInstallHooksContext,  'cocoapods/installer/post_install_hooks_context'
     autoload :PreInstallHooksContext,   'cocoapods/installer/pre_install_hooks_context'
+    autoload :SourceProviderHooksContext, 'cocoapods/installer/source_provider_hooks_context'
```

Comment:
Even without a space they will not be aligned.. Do you want me to rename it so it is shorter?